app/common/public/locales/en/translation.json

```diff
@@ -291,5 +291,6 @@
   "AccessToken": "AccessToken",
   "Token": "Token",
   "Key": "Key",
-  "Secret": "Secret"
+  "Secret": "Secret",
+  "Close tab": "Close tab"
```

Comment:
It is enough to only add English resources. localized resources are synchronized automatically to crowdin as soon as the PR is merged and should not be changed manually.

Reply:
I only kept English resources